zio-http/src/test/scala/zhttp/service/ClientSpec.scala

```diff
@@ -43,6 +43,17 @@ object ClientSpec extends HttpRunnableSpec {
       testM("handle connection failure") {
         val res = Client.request("http://localhost:1").either
         assertM(res)(isLeft(isSubtype[ConnectException](anything)))
+      } + testM("test use default user agent") {
+        val url     = zhttp.http.URL.fromString("https://google.com").toOption.get
+        val request = Client.createRequest(url, headers = Headers.empty, addZioUserAgentHeader = true)
```

Comment:
Unit test should work with `Client.request` we don't need to create a new method for this.